### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by ArwaSAO**

### CODE REVIEW

File: part2.kt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+fun main(args: Array<String>) {
+
+    var string1="The quick brown fox jumps over the lazy dog"
+    var string2= "brown fox jumps"
+
+    println(string2.split(string1))
```
Comment (MohamedFouad96) [inline part2.kt]: try to use substring here

File: part1.kt [hunk 2]
```diff
@@ -0,0 +1,14 @@
+
+fun main(args: Array<String>) {
+   fun getLength(str: String?) : Int {
+    if(str != null)
```
Comment (MohamedFouad96) [inline part1.kt]: There is a problem here, try to correct it

Comment (ArwaSAO) [reply part1.kt]: fun main(args: Array<String>) {
    fun getLength(str: String?) : Int {
        if(str != null)

            return (str. length)
        else
            return 0
    }

    fun main(args: Array<String>) {
        println(getLength("This is a string!"))
    }
}

File: part2.kt (hunk 1 above)
Comment (ArwaSAO) [reply part2.kt]: fun main(){

    var old =" The quick brown fox jumps over the lazy dog."

    var new : String

    println(old.substring(11,26))
}

**Comment by MohamedFouad96** [comment]
Excellent arwa

**Pull request closed without merging**